samples/spring-framework-petclinic/deployApp.ps1

```diff
@@ -0,0 +1,21 @@
+[CmdletBinding()]
+Param(
+	[Parameter(Mandatory=$True, HelpMessage="Example: perftest1. This will create a web app http://perftest1.azurewebsites.net")]
+	[string]$webAppName,
+
+	[Parameter(Mandatory=$True, HelpMessage="Example: myrepo/tomcat:8.5-jre8")]
+	[string]$containerName
+)
+
+$env:RESOURCEGROUP_NAME='tomcat-perftest'
+$env:WEBAPP_NAME=$webAppName
+$env:WEBAPP_PLAN_NAME=$webAppName # use the same name for the web app and app service plan
+$env:CONTAINER_NAME=$containerName
+$env:REGION='westus'
+
+Write-Host -ForegroundColor Green "Deploying to '$env:WEBAPP_NAME'"
+Write-Host -ForegroundColor Green "Container Name: '$env:CONTAINER_NAME'"
+Write-Host -ForegroundColor Green "Resource group: '$env:RESOURCEGROUP_NAME'"
+
+mvn package

```

Comment:
Instead of check-in the code of the sample app here. can you just clone their repo ([https://github.com/spring-projects/spring-petclinic](https://github.com/spring-projects/spring-petclinic)) and use maven to build, package and deploy?

Reply:
pom.xml has changes. Without checking it in here, there will be a manual step.

Reply:
IMO a self-contained sample is better than composing it on the fly. But if you feel strongly I can make the change.

Reply:
@jvano : let me know what you think